portfolio/src/main/webapp/job-experience/job-experience.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+  <link href="https://fonts.googleapis.com/css2?family=Cookie&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="../default.css">
+    <link rel="stylesheet" href="../buttons.css">
+    <link rel="stylesheet" href="job-experience.css">
+  </head>
+  <body>
+    <div id="description-placeholder">
+      <div class="description">
+        Hover over the image to read the description!
+      </div>
+    </div>
+    <div>
+      <div class="img-bubble" id="oij-bubble">
+        <img src="images/oij-logo.svg" class="bubble-content"></img>
+      </div>
+      <div class="img-bubble-big">
+        <div class="description">
+          I'm part of jury comitee of Junior Olympiad in Informatics.
+          I prepare tasks and create tests set for competitors.
+        </div>
+      </div>
+    </div>
+
+    <div>
+      <div class="img-bubble" id="samsung-bubble">
+        <img src="images/samsung-logo.svg" class="bubble-content"></img>
```

Comment:
nit: in HTML5 it's not correct to close the `<img>` tag.

Reply:
Done.